VictoriaBankFinalBeta.rb

```diff
@@ -0,0 +1,96 @@
+require 'watir'
+require 'webdrivers'
+require 'nokogiri'
+require 'C:\Users\olegb\RubymineProjects\untitled\accounts'
+require 'C:\Users\olegb\RubymineProjects\untitled\transactions'
+
+
```

Comment:
This code should work for different users.
You should input credentials from the console.


```ruby
puts "write login"
login = gets.chomp.to_s
puts "write password"
psw = gets.chomp.to_s
```